data/ecosystems/i/iotex.toml

```diff
@@ -1085,3 +1124,126 @@ url = "https://github.com/machinefi/w3bstream_compilets_demo"
 
 [[repo]]
 url = "https://github.com/machinefi/waas-flow-sdk-doc"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-client-python"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-client-go"
+
+[[repo]]
+url = "https://github.com/machinefi/western-chain-ws"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-client-js"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-client-arduino"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-docs-gitbook"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-client-esp32"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-http-client-simulator"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-v2"
+
+[[repo]]
+url = "https://github.com/machinefi/psa-crypto-arduino"
+
+[[repo]]
+url = "https://github.com/machinefi/psa-crypto-esp32"
+
+[[repo]]
+url = "https://github.com/machinefi/create-ws-app"
+
+[[repo]]
+url = "https://github.com/machinefi/web3-iot-sdk"
+
+[[repo]]
+url = "https://github.com/machinefi/google-fit-demo"
+
+[[repo]]
+url = "https://github.com/machinefi/oura-demo-v2"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-vscode-extension-assemblyscript"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-vscode-extension-go"
+
+[[repo]]
+url = "https://github.com/machinefi/w3bstream-vscode-extension-rust"
```

Comment:
might be worth creating a new eco for "MachineFi Lab" and have its org link added to its github_organization field. And then add that new eco as a sub eco to IoTeX. That way we always take all existing and new repos of that org and attribute to IoTeX, wdyt?